app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    def wrapper(*args, **kwargs) -> Any:
```

Comment:
I didn't tell that `key` variable was unneeded 🤐 It was fine, just naming was not really descriptive
If you want your wrapper to get kwargs, you should store them also in dictionary key, like you did before
So you can remove kwargs or store them like in previous solution to make you decorator work correctly in such cases